.github/workflows/gh-pages.yml

```diff
@@ -16,29 +16,34 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
 
-      - name: Build the container and the docs inside of it
-        env:
-          TX_TOKEN: ${{ secrets.TX_TOKEN }}
-        run: |
-          docker build . --build-arg tx_token=$TX_TOKEN -t qfield-docs
-          mkdir site
-          docker run -v "${PWD}/site:/opt/app/site" qfield-docs
-          sudo chown -R "${USER}:${USER}" site
-
       - uses: actions/setup-python@v2
         with:
           python-version: 3.9  # transifex-client has compatibility problem with 3.10
 
-      - name: Install requirements
+      - name: Install Python requirements
         run: pip install -r requirements.txt
 
-      - name: Install requirements insiders
+      - name: Install Python requirements insiders
         run: pip install -r requirements-insiders.txt
         env:
           MKDOCS_INSIDERS_TOKEN: ${{ secrets.MKDOCS_INSIDERS_TOKEN }}
 
+      - name: Install Transifex client
+        run: curl -o- https://raw.githubusercontent.com/transifex/cli/master/install.sh | bash
+
       - name: Deploy documentation
-        run: mkdocs gh-deploy -d ./site --force
+        run: |
+          ./tx add --project qfield-documentation --file-filter 'documentation/<project_slug>.<resource_slug>/<lang>.<ext>' remote https://www.transifex.com/opengisch/qfield-documentation/dashboard/
+          ./tx pull --all
+          mkdocs gh-deploy --force
```

Comment:
This doesn't need the `TX_TOKEN` --> Move to separate step?

Reply:
Hello Matthias, thanks for the comment, so it is only pushing that requires it? their docs was not super explicit about this.

Reply:
`mkdocs gh-deploy --force` does not require the token

Reply:
> `mkdocs gh-deploy --force` does not require the token

Yes, I removed.